### DISCUSSION TITLE
Leistungsverteilung am WR

### DISCUSSION THREAD

**Discussion by edddymax** [hoylabs/OpenDTU-OnBattery#1222] [Q&A] [answered]
Habe ein Zendure -System mit 2Hubs und einen HMS-WR. Mit openDToB kann ich sehr gut die Nulleinspeisung regeln. Gibt es einen Weg gezielt die Leistung der Ackus/PV-Module zu steuern. Hub1 an String 1 und 2 ,Hub2 an String 3 und 4. Aktuell ist eine gleichmäßige Lastverteilung. Dadurch ist natürlich der Acku mit geringerem Ladestand schneller leer. Kann man die Aufteilung prozentual oder Watt-mäßig steuern ? Oder hat jemand eine Idee?

**Comment by spcqike** [comment, marked as answer]
Nein kann man nicht. die Hoymiles belasten die MPPT immer gleich.
